model.py

```diff
@@ -0,0 +1,111 @@
+class Model:
+	def __init__(self):
+		'''
+		Constructor 
+		'''
+		pass
+
+	def user_info_to_database(nutrition_standard,
+                           height,
+                           weight,
+                           age,
+                           meal_list):
+        """
+        Store user info from surveys to database
+        @param
+        nutrition_standard: String
+        height: Int
+        weight: Int
+        age: Int
+        meal_list: List<String>
+
+        Database: UserInfo
+        @column
+        nutrition_standard: String
+        height: Int
+        weight: Int
+        age: Int
+        meal_list: List<String>
```

Comment:
Chắc đây là lần đầu mình động với vấn đề database. Em suggest mình chưa động tới db xịn sò liền, mà hãy lưu dưới dạng binary files. 

Cụ thể, mình dùng [`pickle`](https://docs.python.org/3/library/pickle.html) để lưu bất kì variable nào thành binary file trong máy. Cái này Raghu bày em.

Reply:
a thấy có sqlite3 thấy có vẻ dễ học vs dễ dùng, những cái cơ bản như tạo db thì có thể làm đc. BE xem thử xem

Reply:
okay cái này BE tự design nội bộ nhen